index.bs

```diff
@@ -3090,32 +3090,46 @@ The <dfn export for=commands>browsingContext.setViewport</dfn> command  specific
 
 The [=remote end steps=] with |command parameters| are:
 
-1. Let |context id| be the value of the |command parameters|["<code>context</code>"] field.
+1. Let |context id| be the value of the <code>context</code> field of |command
+   parameters|.
 
-1. Let |context| be the result of [=trying=] to [=get a browsing context=] with |context id|.
+1. Let |context| be the result of [=trying=] to [=get a browsing context=] with
+   |context id|.
 
-1. If |context| is not a [=top-level browsing context=], return [=error=] with [=error code=] [=invalid argument=].
+1. If |context| is not a [=top-level browsing context=], return [=error=] with
+   [=error code=] [=invalid argument=].
 
 1. Let |viewport| be the |command parameters|["<code>viewport</code>"] field.
 
-1. Let |device pixel ratio| be the |command parameters|["<code>devicePixelRatio</code>"] field.
-
-1. If |viewport| is not null, and the implementation is unable to set the width and height dimensions
-   of the [=layout viewport=] to |viewport|["width"] and |viewport|["height"], respectively, or if 
-   |device pixel ratio| is not null and the implementation is unable to set the size of the CSS pixel 
-   of the [=layout viewport=] to |device pixel ratio| device pixels, or if the implementation is unable
-   to adjust the [=layout viewport=] parameters for any other reason, return [=error=] 
-   with [=error code=] [=unsupported operation=].
+1. Let |device pixel ratio| be the value of the <code>devicePixelRatio</code>
```

Comment:
So, I prefer using the `Let |field| be |command parameters|["<code>field</code>"]` style. 

But also since it can be missing and map lookups are supposed to be infallible, you now need to first check if `|command parameters| [=map/contains=] "<code>field</code>"`.

Reply:
Done, however I think using field accessor notation goes counter to the previous thought that we don't want to lock into a language (like JavaScript). Fields can be accessed in other ways.

Reply:
Hmm, interesting. TIL.